docs/basic-features/eslint.md

```diff
@@ -172,4 +172,24 @@ module.exports = {
 }
 ```
 
-This eliminates any risk of collisions that can occur due to importing the same plugin or parser across multiple configurations.
+The plugin can be installed normally in your project without needing to run `next lint`:
+
+```bash
+npm install --save-dev eslint-plugin-next
+# or
+yarn add --dev eslint-plugin-next
+```
```

Comment:
@leerob I also added these few small lines to explain how the plugin can be installed separately 🙏

Reply:
should this be `@next/eslint-plugin-next`?